<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UMChat</class>
 <widget class="QWidget" name="UMChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="mainFrame">
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="margin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QtChatHistoryWidget" name="chatHistory">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="editFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>47</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>47</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="horizontalSpacing">
          <number>0</number>
         </property>
         <property name="verticalSpacing">
          <number>10</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QFrame" name="chatEditFrame">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>45</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>45</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QFrame" name="chatFunctionFrame">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>48</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>48</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <widget class="ImageButton" name="sendButton" native="true">
            <property name="geometry">
             <rect>
              <x>1</x>
              <y>23</y>
              <width>58</width>
              <height>24</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>58</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>58</width>
              <height>24</height>
             </size>
            </property>
            <property name="text" stdset="0">
             <string/>
            </property>
            <property name="iconSize" stdset="0">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
           </widget>
           <widget class="QFrame" name="buttonFrame">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>60</width>
              <height>21</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>21</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>21</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <widget class="ImageButton" name="btnTranslation" native="true">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>3</y>
               <width>13</width>
               <height>18</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>13</width>
               <height>18</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>13</width>
               <height>18</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Universal Translator</string>
             </property>
             <property name="text" stdset="0">
              <string/>
             </property>
            </widget>
            <widget class="ImageButton" name="btnEmoticon" native="true">
             <property name="geometry">
              <rect>
               <x>23</x>
               <y>3</y>
               <width>14</width>
               <height>18</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>14</width>
               <height>18</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>14</width>
               <height>18</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Insert Emoticon</string>
             </property>
             <property name="text" stdset="0">
              <string/>
             </property>
            </widget>
            <widget class="ImageButton" name="btnFont" native="true">
             <property name="geometry">
              <rect>
               <x>6</x>
               <y>3</y>
               <width>13</width>
               <height>18</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>13</width>
               <height>18</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>13</width>
               <height>18</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Modify Font</string>
             </property>
             <property name="text" stdset="0">
              <string/>
             </property>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
  <customwidget>
   <class>QtChatHistoryWidget</class>
   <extends>QWebView</extends>
   <header>chat/QtChatHistoryWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ImageButton</class>
   <extends>QWidget</extends>
   <header>qtutil/ImageButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../resources/wengophone.qrc"/>
 </resources>
 <connections/>
</ui>
